lib/profile-onboarding.dart

```diff
@@ -96,58 +109,88 @@ class _ProfileOnboardingState extends State<ProfileOnboarding> {
           ),
           onSaved: onSaved,
         ),
-        SizedBox(height: 20.0),
+        _sizedBoxPadding(),
       ],
     );
   }
 
-  DateTimePicker birthDateSelector() {
+  DateTimePicker _birthDateSelector() {
     return DateTimePicker(
       initialValue: '',
       firstDate: DateTime(1930),
       lastDate: DateTime(2030),
       decoration: InputDecoration(
         border: OutlineInputBorder(),
-        labelText: 'Birthday',
+        labelText: 'Date of Birth',
```

Comment:
The calendar popping up and having to either scroll up for your birthdate or editing it seems unnecessary. I would just have the user enter in their birthdate in the form: 01/20/2020.

Reply:
Other than that, everything else works and looks good!